posthog/models/notebook/notebook.py

```diff
@@ -12,6 +12,7 @@ class Notebook(UUIDModel):
     team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
     title: models.CharField = models.CharField(max_length=256, blank=True, null=True)
     content: JSONField = JSONField(default=None, null=True, blank=True)
+    text_content: models.TextField = models.TextField(blank=True, null=True)
```

Comment:
Goes without saying that long term this is probably not scalable... Sending the full JSON every time on update is already not great, sending an additional field full of text with it...

But I digress...

Reply:
My assumption here is that we don't want to build and maintain python parsing for tip taps data model

My experience with CMSs is that it's pretty standard to hold a separate representation of the data only to support text search (and my gut is that its simpler)

There's nothing stopping us patching these two fields instead of replacing them (other than time and understanding 🤣)

Reply:
I tell a lie

> There's nothing stopping us patching 

There is... this unstructured blob can't remove text and has to be replaced entirely